Acornfile

```diff
@@ -34,6 +37,7 @@ jobs: "create-resources": {
     DB_NAME: args.dbName
     PROVIDER: args.provider
     REGION: args.region
+    SUBSCRIPTION_ID: args.subscriptionId
   }
   events: ["create"]
```

Comment:
We need to do "create" and "update" events. If the user messes something up with an id or plan name or region any number of thing, they have to delete and restart because the job won't run again. Otherwise, there should be an update job that does intelligent things.

Reply:
I think using a dedicated update job is a good option as it's easy to give this one (via the *resources* secret) the subscription and database ids resulting of the create job. Splitting the creation / update logic in 2 distinct jobs makes a lot of sense.